tensorflow_datasets/core/dataset_builder.py

```diff
@@ -483,14 +516,14 @@ def as_dataset(self,
     return datasets
 
   def _build_single_dataset(
-      self,
-      split,
-      shuffle_files,
-      batch_size,
-      decoders,
-      read_config,
-      as_supervised,
-      in_memory):
+    self,
```

Comment:
Please use https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style to correctly indent the code.

Reply:
Done